R/mod_chart_example.R

```diff
@@ -40,24 +38,24 @@ mod_chart_example_server <- function(id) {
     ns <- session$ns
 
     output$chart <- highcharter::renderHighchart({
-
+      
       # Generate bins based on input$bins from ui.R
       x <- nhsbsaShinyR::faithful[, 2]
       bins <- seq(min(x), max(x), length.out = as.numeric(input$bins) + 1)
 
       # Draw the histogram with the specified number of bins
-      chart <- hist(x, breaks = bins)
+      chart <- graphics::hist(x, breaks = bins, plot = FALSE)
```

Comment:
Namespace needed as even tho `graphics` is a package part of the base installation of R, it is not the `base` package itself.

Reply:
Also, without `plot = FALSE` did not work for me...